README.md

```diff
@@ -1,15 +1,15 @@
 # :warning: Announcements 
 
-Effective September 15th, 2023, the Focalboard plugin will no longer be bundled with Mattermost and will transition to being fully community supported. 
+Effective September 15th, 2023, the Focalboard plugin is no longer bundled with Mattermost and has transitioned to being fully community supported. 
```

Comment:
propose removing this and moving down to minimize duplication with the legal statement